client/src/InfoBase/App.js

```diff
@@ -162,11 +165,12 @@ export class App extends React.Component {
               />
               <Route path="/rpb/:config?" component={ReportBuilder} />
               <Route path="/about" component={About} />
+              <Route path="/faq/:selected_qa_key?" component={Faq} />
               <Route
                 path="/compare_estimates/:h7y_layout?"
                 component={EstimatesComparison}
               />
-              <Route path="/privacy" component={PrivacyStatement} />
+              <Route path="/privacy" component={PrivacyStatement} />s
```

Comment:
I didn't realize routes supported pluralization. Is that french friendly, though? 😆

Reply:
Frick, always screwing up my quick save hotkey, good catch!